src/methods.js

```diff
@@ -179,7 +179,7 @@ export default {
     features: {
       multiwallet: '>=0.15.0'
     },
-    version: '>=0.11.0'
```

Comment:
Sorry about the misinformation in [here](https://github.com/ruimarinho/bitcoin-core/pull/100#issuecomment-532198709) but revert this change because `generate` is only deprecated (not removed).